sentinelsat/scripts/cli.py

```diff
@@ -231,6 +239,11 @@ def cli(
         with open(os.path.join(path, "search_footprints.geojson"), "w") as outfile:
             outfile.write(gj.dumps(footprints_geojson))
 
+    if location is not None:
+        query_answers = placename_to_wkt(location)
+        click.echo("The location we are querying is: '{}'".format(query_answers[1]))
```

Comment:
I would also print the WKT envelope string in addition to the place name to make it clearer what the area being queried actually is and how big it is.